libraries/helper.rb

```diff
@@ -46,14 +46,24 @@ def omnibus_env
       node.run_state[:omnibus_env] ||= Hash.new { |hash, key| hash[key] = [] } # ~FC001
     end
 
+    # TODO: this method should just go away
     def omnibus_toolchain_enabled?
-      # We don't have an omnibus toolchain for windows yet.
-      !windows?
+      true
+    end
+
+    def toolchain_install_dir
+      if windows?
+        windows_safe_path_join(ENV['SYSTEMDRIVE'], 'opscode', node['omnibus']['toolchain_name'])
```

Comment:
So, the reasoning I think is that C:\chef is currently the config directory for chef and angry-chef on Windows (https://github.com/chef/chef/blob/master/omnibus/resources/chef/msi/source.wxs.erb#L117-L122). We'll need to change that before we can really do anything else.